src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private double firstLeg;
+    private double secondLeg;
+
+    public RightTriangle(double firstLeg, double secondLeg) {
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+
+    }
+
+    @Override
+    public double calculateArea() {
+        return 0.5 * firstLeg * secondLeg;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println(this);
+    }
+
+    @Override
+    public String toString() {
+        final StringBuilder sb = new StringBuilder("Rectangle{");
```

Comment:
There is a mistake in the `toString()` method. The class name should be `RightTriangle` instead of `Rectangle`.